static/app/views/replays/detail/network/utils.tsx

```diff
@@ -50,17 +50,14 @@ export function sortNetwork(
 
 export const getResourceTypes = (networkSpans: NetworkSpan[]) =>
   Array.from(
-    new Set<string>(
-      networkSpans.map(networkSpan => networkSpan.op.replace('resource.', '')).sort()
-    )
-  );
+    new Set(networkSpans.map(networkSpan => networkSpan.op.replace('resource.', '')))
+  ).sort();
```

Comment:
sort the de-duplicated array, and we don't need to be so specific with the types because latest TS is smarter!